pkg/cmd/server/kubernetes/master/authorizer.go

```diff
@@ -0,0 +1,55 @@
+package master
+
+import (
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
+	authorizerunion "k8s.io/apiserver/pkg/authorization/union"
+	kinformers "k8s.io/client-go/informers"
+	"k8s.io/kubernetes/pkg/auth/nodeidentifier"
+	kinternalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
+	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/node"
+	rbacauthorizer "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac"
+	kbootstrappolicy "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
+
+	openshiftauthorizer "github.com/openshift/origin/pkg/authorization/authorizer"
+	"github.com/openshift/origin/pkg/authorization/authorizer/browsersafe"
+	"github.com/openshift/origin/pkg/authorization/authorizer/scope"
+)
+
+func NewAuthorizer(internalInformers kinternalinformers.SharedInformerFactory, informers kinformers.SharedInformerFactory, projectRequestDenyMessage string) authorizer.Authorizer {
+	messageMaker := openshiftauthorizer.NewForbiddenMessageResolver(projectRequestDenyMessage)
```

Comment:
@deads2k can we merge #20379 first so this forbidden message maker can die before moving here?

That PR is just waiting on a LGTM from @smarterclayton

Reply:
> @deads2k can we merge #20379 first so this forbidden message maker can die before moving here?
> 
> That PR is just waiting on a LGTM from @smarterclayton

Get my comment fixed and I'll do it.

Continue review here please.